src/recon_buildblock/ProjMatrixByBinUsingRayTracing.cxx

```diff
@@ -676,15 +678,19 @@ calculate_proj_matrix_elems_for_one_bin(
 
       phi = proj_data_info_noarccor.get_phi(bin);
       s_in_mm = proj_data_info_noarccor.get_s(bin);
+      tantheta = proj_data_info_noarccor.get_tantheta(bin);
     }
 
   }
   
   const float cphi = cos(phi);
   const float sphi = sin(phi);
   
-  const float tantheta = proj_data_info_ptr->get_tantheta(bin);
-  const float costheta = 1/sqrt(1+square(tantheta));
+
+  float costheta = 1/sqrt(1+square(tantheta));
+  if (costheta < 0.0001) {
+    costheta = 0.0001;
+  }
```

Comment:
can't see that we can ever have this (an LOR parallel to the scanner axis), but ok. do we need to threshold? Maybe add a comment why you do this

Reply:
hmm it's really that simple... do you want me to commit this change?

Reply:
ok for me.

@danieldeidda, do we have a test on `get_tantheta` in `test_proj_data_info` already? (easiest one: compare the cylindrical and blocks give the same result for the identical scanner).